Makefile

```diff
@@ -27,11 +27,17 @@ cover:
   find tests -type d | xargs nosetests --with-coverage --cover-inclusive --cover-erase --cover-package=twilio
 
 docs-install:
-	. venv/bin/activate; pip install twilio sphinx
+	. venv/bin/activate; pip install -U sphinx
 
 docs:
-	. venv/bin/activate; sphinx-apidoc -f twilio -o docs/source
-	cd docs && make clean && make html
+	-rm -rf source/_rst/*
+	-rm -rf build/*
+	. venv/bin/activate; sphinx-apidoc -f twilio -o docs/source/_rst
+	. venv/bin/activate; sphinx-build -b html -c ./docs -d docs/build/doctrees   . docs/build/html
```

Comment:
how does sphinx-build know where to find index.rst?

Reply:
It's on `conf.py` and the -c flag indicates where the conf file lives